lib/adaptors/serialport.js

```diff
@@ -52,13 +52,13 @@ util.inherits(Adaptor, EventEmitter);
  */
 Adaptor.prototype.open = function open(callback) {
   var self = this,
-      port = this.serialport = new serialport.SerialPort(this.conn, {});
+      port = this.serialport = new serialport.SerialPort(this.conn, {}, false);
 
   function emit(name) {
     return self.emit.bind(self, name);
   }
 
-  port.on("open", function(error) {
+  port.open(function(error) {
```

Comment:
@olcar is exactly correct, Serialport fires `open` event if/when the port is opened after it has been created. Why change this?